### PR TITLE
Fv and transport

### PR DESCRIPTION
**Pull request by IvarStefansson**
Some adjustments on the node elimination.
Completed (for now) the FV flux computation for grid buckets (also with eliminated nodes).
Updated the upwind coupling accordingly (discretization + time step estimate).
Significant additions to testing of FV code and some examples.

### CODE REVIEW

File: src/porepy/grids/grid_bucket.py [hunk 1]
```diff
@@ -399,7 +416,7 @@ def edge_prop(self, grid_pairs, key):
         for gp in np.atleast_2d(grid_pairs):
             if tuple(gp) in self.graph.edges():
                 prop_list.append(self.graph.edge[gp[0]][gp[1]][key])
-            elif tuple(gp[::-1]) in self.graph.edges():
+            elif tuple(gp[::-1]) in self.graph.edges():
```
Comment (keileg) [inline src/porepy/grids/grid_bucket.py]: Remove extra blankspace?

File: src/porepy/grids/grid_bucket.py [hunk 2]
```diff
@@ -653,15 +670,17 @@ def copy(self):
 
 #------------------------------------------------------------------------------#
 
-    def duplicate_without_dimension(self, dim):
+    def duplicate_without_dimension(self, dim, compute_new_fluxes=False):
```
Comment (keileg) [inline src/porepy/grids/grid_bucket.py]: I don't like the mixture of grids and numerics. Can it be avoided?

File: src/porepy/grids/grid_bucket.py [hunk 3]
```diff
@@ -671,61 +690,74 @@ def find_shared_face(self, n1, n2, zero_d_node):
...
         for i in range(n_neighbors - 1):
             n1 = neighbors[i]
             for j in range(i + 1, n_neighbors):
                 n2 = neighbors[j]
-                face_faces = self.find_shared_face(n1, n2, node)
-                self.add_edge([n1, n2], face_faces)
-
+                face_faces, cell_cells = self.find_shared_face(n1, n2, node)
+                self.add_edge([n1, n2], cell_cells)
+                if compute_new_fluxes:
```
Comment (keileg) [inline src/porepy/grids/grid_bucket.py]: What happens here?

File: examples/example1/benchmark1/fv_blocking.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        bc_val[bound_faces[right]] = np.ones(np.sum(right))
+                               
+        param.set_bc('flow', bc.BoundaryCondition(g, bound_faces, labels))
+        param.set_bc_val('flow', bc_val)
+
+        d['param'] = param
+
+#------------------------------------------------------------------------------#
+
+multi_point = False
```
Comment (IvarStefansson) [inline examples/example1/benchmark1/fv_blocking.py]: No, could be done.

**Pull request merged**